addons/website_event/tests/common.py

```diff
@@ -32,9 +32,9 @@ def setUpClass(cls):
         })
 
     def _get_menus(self):
-        return {'Introduction', 'Community', 'Info', 'Location'}
+        return {'Home', 'Practical', 'Rooms'}
 
-    def _assert_website_menus(self, event, menus_in=None, menus_out=None):
+    def _assert_website_menus(self, event, menus_in=None, menus_out=None, child_menus_in=None, child_menus_out=None):
```

Comment:
Not quite what I meant :smile: 

I think we can keep it simple with the old signature.

But here below, you'd do:
```py
menus = self.env['website.menu'].search([('parent_id', '=', event.menu_id.id)])
menus |= menus.child_ids  # add child menus to simplify checks, containing notably talks submenus
```